### PR TITLE
[SYTO-1151] Cannot control color of several icons

### PR DESCRIPTION
**Pull request by hkekeli**
https://jira.natgeo.com/browse/SYTO-1151

The following icons should change color they have a color class applied:
menu-down
menu-up
menu
search
social-google-plus
social-instagram

to test, apply `mt3-color--primary` to the icons 

iconcolor

### CODE REVIEW

File: src/icons/menu-down.svg [hunk 1]
```diff
@@ -1,6 +1 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Generator: Adobe Illustrator 20.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 viewBox="0 0 36 34.3" style="enable-background:new 0 0 36 34.3;" xml:space="preserve">
-<polygon points="0,8.2 18,26.1 36,8.2 "/>
-</svg>
+<svg id="Layer_1" style="fill: currentColor" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 23.87"><title>Artboard 1</title><polygon points="0 0 24 23.87 48 0 0 0"/></svg>
```
Comment (alexrqs) [inline src/icons/menu-down.svg]: you are using Title Artboard 1 for all the components
https://developer.mozilla.org/en/docs/Web/SVG/Element/title

Comment (alexrqs) [inline src/icons/menu-down.svg]: also as the id attribute on html should be unique please change the id="Layer_1" to something more accurate

Comment (alexrqs) [inline src/icons/menu-down.svg]: same for data-name

Comment (hkekeli) [inline src/icons/menu-down.svg]: @alexrqs We're not handcoding these svgs.

Comment (alexrqs) [inline src/icons/menu-down.svg]: ok

**Comment by krlospelaez** [comment]
@hkekeli I think we don't need to change the svg icons, just adding `fill: currentColor` inside `mt3_icon` rule should works. Please update the PR.

**Comment by krlospelaez** [comment]
LGTM 👍

**Pull request merged**